manuscript/100_Analysis_In_TDD.md

```diff
@@ -42,7 +42,7 @@ Gherkin
 
 Hungry? Too bad, because the Gkerkin I�am gonna talk about is not edible. It is a�notation and a�way of thinking about behaviors of the specified piece of code. It can be applied on different levels of granularity -- any behavior, whether of a�whole system or a�single class, may be described using it.
 
-We actually talked about Gherkin before in this book, just did not name it. It is the GIVEN-WHEN-THEN structure that you can see everywhere, even in code samples as comments. This time, we are stamping a�name on it and analyzing it further.
+We talked about Gherkin before in this book, just did not name it. It is the GIVEN-WHEN-THEN structure that you can see everywhere, even in code samples as comments. This time, we are stamping a�name on it and analyzing it further.
```

Comment:
Changed again...:
"In fact we already used the Gherkin notation, we just didn't name it so."